ext/awscrt.stub.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+/** @generate-class-entries */
+
+function aws_crt_last_error(): int {}
+
+function aws_crt_error_name(int $error_code): string {}
+
+function aws_crt_error_str(int $error_code): string {}
+
+function aws_crt_error_debug_str(int $error_code): string {}
```

Comment:
trivial: probably don't need to bind `aws_crt_error_debug_str()`, it's just a single string that smooshes together the info from the other 3. Chances are you'll package those 3 bits of info together in an exception class

Reply:
Yeah, that's fine, it's just in the API, and I'm going for blind full coverage of everything that isn't part of the testing stuff.